backend_core/config/settings.py

```diff
@@ -145,6 +145,13 @@
 
 AUTH_USER_MODEL = 'users.CustomUser'
 
+LOGGING = {
```

Comment:
Po proszę to zabezpieczyć, aby działało, tylko na trybie debug, gdyż jest ryzyko, że obciąży dosyć mocno serwer

Reply:
Można tez ustawić enva, DJ_DB_LOGGING i w zależności od tego mogłaby działać ta funkcjonalność

Reply:
Poprawiłem, proszę o CR